pkg/controller/policyrecommendation/policyrecommendation_controller.go

```diff
@@ -417,20 +394,38 @@ func (r *ReconcilePolicyRecommendation) Reconcile(ctx context.Context, request r
 
 		certificateManager.AddToStatusManager(r.status, helper.InstallNamespace())
 
-		policyRecommendationCfg.TrustedBundle = trustedBundle
-		policyRecommendationCfg.PolicyRecommendationCertSecret = policyRecommendationKeyPair
-
-		components = append(components,
-			rcertificatemanagement.CertificateManagement(&rcertificatemanagement.Config{
-				Namespace:       helper.InstallNamespace(),
-				TruthNamespace:  helper.TruthNamespace(),
-				ServiceAccounts: []string{render.PolicyRecommendationName},
-				KeyPairOptions: []rcertificatemanagement.KeyPairOption{
-					rcertificatemanagement.NewKeyPairOption(policyRecommendationKeyPair, true, true),
-				},
-				TrustedBundle: trustedBundle,
-			}),
-		)
+		if !r.multiTenant {
```

Comment:
Can you add a comment here indicating why we have a RW trusted bundle for non-multitenant but RO for multi-tenant? Just to help us in the future.